internal/testutils/cmd.go

```diff
@@ -0,0 +1,29 @@
+package testutils
+
+import (
+	"log"
+	"os/exec"
+	"strings"
+	"time"
+)
+
+// RunWithTiming runs the given command while logging its duration with the provided message.
+func RunWithTiming(msg string, cmd *exec.Cmd) error {
+	log.Printf("%s: %s", msg, strings.Join(cmd.Args, " "))
```

Comment:
I would rely on `t.Logf` here and make these helpers

Reply:
already did that on the other branch. copied it over